### ISSUE TITLE
Add interface to upload own stand data

### ISSUE THREAD

**Issue by lfehrma** [AWF-GAUG/fisimshiny#1]
Would it be possible to add an interface that allows uploading own stand data (e.g. from .csv files with predefined structure)? Using create_pop is not very flexible and it would be nice to be able to add more stands.

**Comment by sebschnell** [comment]
Yes, shouldn't be a bigger problem. Good idea! I'll have a look.

What do you mean with create_pop? I can't find this function anywhere.
